src/main/java/uk/gov/hmcts/reform/pip/account/management/service/PublicationService.java

```diff
@@ -60,6 +61,31 @@ public boolean sendNotificationEmail(String emailAddress, String forename, Strin
         }
     }
 
+    /**
+     * Method which sends a request to the publication-services microservice to send a media account rejection email.
+     *
+     * @param mediaApplication - MediaApplication object containing applicant details
+     * @param reasons          - reasons for rejection
+     * @return boolean for logging success or failure
+     */
+    public boolean sendMediaAccountRejectionEmail(MediaApplication mediaApplication,
+                                                  Map<String, List<String>> reasons) {
+        JSONObject jsonObject = new JSONObject();
+        jsonObject.put("applicantId", mediaApplication.getId().toString());
+        jsonObject.put("fullName", mediaApplication.getFullName());
+        jsonObject.put("email", mediaApplication.getEmail());
```

Comment:
There are already constants called FULL_NAME and EMAIL at the top of these class so can use that instead. This is causing new code smells

Reply:
Have updated